documentation/library_guide/overview.rst

```diff
@@ -73,13 +73,13 @@ Follow the steps below to build your code with |onedpl_short|:
 #. To build with the |dpcpp_cpp|, see the `Get Started with the Intel® oneAPI DPC++/C++ Compiler
    <https://software.intel.com/content/www/us/en/develop/documentation/get-started-with-dpcpp-compiler/top.html>`_
    for details.
-#. Set the environment for |onedpl_short| and |onetbb_short|.
+#. Set the environments for |onedpl_short| and |onetbb_short|.
```

Comment:
```suggestion
#. Set the environment variables for |onedpl_short| and |onetbb_short|.
```
I suggest this wording as in this case the environment should be common for oneDPL and oneTBB.